frontend/src/components/Entity.vue

```diff
@@ -42,6 +43,12 @@ export default {
           icon: "mode_edit",
           tooltip: "Edit/show entity details"
         },
+        {
+          name: "Bulk Add (copy Attributes)",
+          component: EntityBulkAdd,
```

Comment:
This should be wrapped in markRaw or shallowRef (I'm fixing all similar occurences to markRaw in my commit, so I'd prefer markRaw).